comp/otelcol/logsagentpipeline/logsagentpipelineimpl/agent.go

```diff
@@ -207,19 +202,12 @@ func (a *Agent) GetPipelineProvider() pipeline.Provider {
 func (a *Agent) SetupPipeline(
 	processingRules []*config.ProcessingRule,
 ) {
-	health := health.RegisterLiveness("logs-agent")
-
-	// setup the auditor
-	// We pass the health handle to the auditor because it's the end of the pipeline and the most
-	// critical part. Arguably it could also be plugged to the destination.
-	auditorTTL := time.Duration(a.config.GetInt("logs_config.auditor_ttl")) * time.Hour
-	auditor := auditor.New(a.config.GetString("logs_config.run_path"), auditor.DefaultRegistryFilename, auditorTTL, health)
 	destinationsCtx := client.NewDestinationsContext()
 
 	// setup the pipeline provider that provides pairs of processor and sender
 	pipelineProvider := pipeline.NewProvider(
 		a.config.GetInt("logs_config.pipelines"),
-		auditor,
+		&sender.NoopSink{},
```

Comment:
cc @liustanley is this what we want?

Reply:
Yes this is what we want, the auditor is used for log tailers and the log agent's liveness check. We don't use either of these in DDOT so this no-op implementation is correct